integration/system/options_test.go

```diff
@@ -0,0 +1,29 @@
+package system
+
+import (
+	"net/http"
+	"testing"
+
+	req "github.com/docker/docker/internal/test/request"
+	"gotest.tools/assert"
+)
+
+func TestAPIOptionsRoute(t *testing.T) {
+	defer setupTest(t)()
+
+	resp, _, err := req.Do("/", req.Method(http.MethodOptions))
+	assert.NilError(t, err)
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+}
+
+func TestAPIGetEnabledCORS(t *testing.T) {
+	resp, body, err := req.Get("/version")
+	assert.NilError(t, err)
+	defer body.Close()
+	assert.Equal(t, resp.StatusCode, http.StatusOK)
+	// TODO: runcom incomplete tests, why old integration tests had this headers
+	// and here none of the headers below are in the response?
+	//c.Log(res.Header)
+	//c.Assert(res.Header.Get("Access-Control-Allow-Origin"), check.Equals, "*")
+	//c.Assert(res.Header.Get("Access-Control-Allow-Headers"), check.Equals, "Origin, X-Requested-With, Content-Type, Accept, X-Registry-Auth")
```

Comment:
LGTM, but probably these old comments can be removed?

Reply:
@AkihiroSuda Updated, thanks.